app/models.py

```diff
@@ -1,19 +1,12 @@
 from app import db
-from flask_script import Manager
-from flask_migrate import Migrate, MigrateCommand
 
-
-migrate = Migrate(app, db)
-manager = Manager(app)
-
-manager.add_command('db', MigrateCommand)
 class Day(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     temperature = db.Column(db.Float, nullable=False)
     date = db.Column(db.String(20), nullable=False)
-    high_risk = db.Column(db.Boolean(), nullable=True)
+    high_risk = db.Column(db.Boolean, nullable=True)
 
-    def __init__(self, temperature, date):
+    def __init__(self, temperature, date, high_risk):
```

Comment:
now that I'm seeing this I dont think we need high risk here

Reply:
yeah, that makes sense to add that attribute ourselves after we do some calculations